### PR TITLE
fix(BS-15117) Custom permissions mapping and profiles with spaces

### PR DESCRIPTION
**Pull request by abirembaut**
Special characters like white space must be replaced with their unicode value (For example \u0020 for the white space)
Add unit test and guidance in the properties files

covers [BS-15117](https://bonitasoft.atlassian.net/browse/BS-15117)

### CODE REVIEW

File: common/src/test/java/org/bonitasoft/console/common/server/preferences/properties/SimplePropertiesTest.java [hunk 1]
```diff
@@ -50,6 +50,16 @@ public void should_getProperty_return_the_right_custom_permissions() throws Exce
     }
 
     @Test
+    public void should_getProperty_return_the_right_custom_permissions_with_special_characters() throws Exception {
```
Comment (cpuy) [inline common/src/test/java/org/bonitasoft/console/common/server/preferences/properties/SimplePropertiesTest.java]: This is not an English sentence ! Lets read that:
> SimpleProperties should getProperty return the right custom permissions with special characters

hum......
should be `should_return_the_right_custom_permissions_with_special_characters`

**Comment by cpuy** [comment]
LGTM. please see small note above

File: common/src/test/java/org/bonitasoft/console/common/server/preferences/properties/SimplePropertiesTest.java (hunk 1 above)
Comment (abirembaut) [inline common/src/test/java/org/bonitasoft/console/common/server/preferences/properties/SimplePropertiesTest.java]: I agree. Copy-Paste from another test...

**Comment by abirembaut** [comment]
Fixed

**Comment by cpuy** [comment]
:+1: 
In fact we don't want to write `getProperty_should_blablba` because what is under test is not a method but an object.
So tests should be read as follow `SimpleProperties should_return_the_right_custom_permissions_with_special_characters`. It enforce us to use proper names on objecr we design.
Anyway, not a big deal

**Pull request merged**